### PR TITLE
[ROS monitor] In favor of vscode color scheme over hard-coded.

### PR DESCRIPTION
**Pull request by seanyen**
Be in favor of `vscode` color scheme over hard-coded.

### CODE REVIEW

**Comment by kejxu** [comment]
would it be possible to add screenshots of the page with the change in effect?

File: assets/ros/core-monitor/style.css [hunk 1]
```diff
@@ -8,7 +8,7 @@ table {
 }
 
 table thead {
-  background: #333;
+  background: var(--vscode-sideBarSectionHeader-background);
```
Comment (seanyen) [inline assets/ros/core-monitor/style.css]: I make use of the color from the background of sidebar header.
image

**Comment by seanyen** [comment]
> would it be possible to add screenshots of the page with the change in effect?

Sure. Here is the light & dark theme.
image

image

File: assets/ros/core-monitor/style.css [hunk 2]
```diff
@@ -20,11 +20,11 @@ table td, table th {
 }
 
 table tbody tr {
-  border-bottom: 1px solid #333;
+  border-bottom: 1px solid var(--vscode-sideBarSectionHeader-background);
 }
 
 table tbody tr:hover {
```
Comment (kejxu) [inline assets/ros/core-monitor/style.css]: it seems we might be removing the hover effect if we use the same color, but this is minor

**Pull request merged**